.circleci/config.yml

```diff
@@ -202,6 +221,7 @@ jobs:
           username: $DOCKER_USER
           password: $DOCKER_PASS
     steps:
+      - check-for-deployment
```

Comment:
The `check-for-deployment` command, sends an API call to CircleCI that will halt subsequent jobs. Including it here, in `docker-image-publish-stage`, will mean the load test image still gets deployed, which I think is unexpected.

I suggest to have the command in an independent job, Instead of including this command as a step in this `docker-image-publish-stage` job. For instance how it's currently done in Merino:

<img width="1274" alt="image" src="https://github.com/mozilla-services/contile/assets/3422688/47ac303c-1bdf-4a76-8904-421afea06bda">

Reply:
Sounds good. I redefined the logic to `skip-if-do-not-deploy` as the command and then `check-for-deployment` as the actual job called in the workflow, like Merino.